shop-ui/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="text_view_category_error_header">Something went wrong</string>
```

Comment:
What if we need to use it not only in text view? Please rename prefix to `label`

Reply:
we don't need it now, I just deleted these lines